src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,74 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final String INFORMATION_SEPARATOR = ",";
+    private static final String SUPPLY_OPERATION = "supply";
+    private static final String BUYING_OPERATION = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(fromFileName);
+        StringBuilder stringBuilder = new StringBuilder();
+        int buy = 0;
+        int supply = 0;
+        int resultOperation = 0;
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(file));
+            int value = reader.read();
+            while (value != -1) {
+                stringBuilder.append((char) value);
+                value = reader.read();
+            }
+            String[] operationsWithGoods = stringBuilder.toString().split(System.lineSeparator());
+            for (String operation: operationsWithGoods) {
+                String[] splitOperation = operation.split(INFORMATION_SEPARATOR);
+                if (isSupply(splitOperation)) {
+                    supply += Integer.parseInt(splitOperation[1].replaceAll("[^0-9]", ""));
+                }
+                if (isBuy(splitOperation)) {
+                    buy += Integer.parseInt(splitOperation[1].replaceAll("[^0-9]", ""));
+                }
+            }
+            fileBuilder(statisticBuilder(supply, buy),toFileName);
```

Comment:
please use formatting code in the IDE before to commit to git. it is common practice. space should be present after coma.